tests/runner.py

```diff
@@ -1540,7 +1540,7 @@ def btest(self, filename, expected=None, reference=None, force_c=False,
     if 'WASM=0' not in args:
       # Filter out separate-asm, which is implied by wasm
       args = [a for a in args if a != '--separate-asm']
-    args += ['-DEMTEST_PORT_NUMBER=%d' % self.port, '-include', path_from_root('tests', 'report_result.h')]
+    args += ['-DEMTEST_PORT_NUMBER=%d' % self.port, '-include', path_from_root('tests', 'report_result.h'), path_from_root('tests', 'report_result.cpp')]
```

Comment:
Adding a .cpp file as a compiler flag seems a little off.

Reply:
I added a comment and cleaned this up a little - what do you think now?

I think it's ok here to add another source file to be compiled as another argument, with a comment explaining why, but maybe I don't understand what you don't like about it? It basically ends up with `emcc [original source files and arguments] another_source.cpp`, so it adds it to the existing sources.